.github/workflows/main.yml

```diff
@@ -538,23 +658,86 @@ jobs:
         name: surelog-windows-${{ matrix.compiler }}
         path: ${{ github.workspace }}/build/surelog-windows-${{ matrix.compiler }}.tar.gz
 
+    - name: Package up binary for regression
+      shell: bash
+      run: |
+        tar cvzf build/surelog-binary.tar.gz \
+                build/bin/surelog* build/bin/roundtrip* build/bin/pkg \
+                build/third_party/UHDM/bin
+
+    - name: Upload Binary for regression
+      uses: actions/upload-artifact@v3
+      with:
+        name: build-result-windows-${{matrix.compiler}}
+        path: build/surelog-binary.tar.gz
```

Comment:
Probably it would work. I am not too familiar with the regression.py script, I suspected it would look in `build/` for the binaries. But if we can run the regression from the install artifacts the better.